src/components/InlineDialog/InlineDialog.stories.tsx

```diff
@@ -0,0 +1,302 @@
+        },
+        'data-test-id': {
+            type: 'string',
+        },
+        role: {
+            type: 'string',
+        },
+        darkUnderlay: {
+            type: 'boolean',
+        },
+        autoHeight: {
+            type: 'boolean',
+        },
+        zIndex: { table: { disable: true } },
+    },
+} as Meta<InlineDialogProps>;
+
+const TextExample = () => {
+    return (
+        <p className="tw-my-2">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Distinctio fugiat hic incidunt, inventore odio
+            praesentium qui quisquam! Dignissimos nemo quisquam unde voluptatem. Animi ea iusto numquam odio omnis.
+            Dicta, voluptatum.
+        </p>
+    );
+};
+
+const Template: StoryFn<InlineDialogProps> = (args) => {
+    const [isOpen, setIsOpen] = useToggleOverlay(false, { isBlockingModal: args.modality === Modality.BlockingModal });
+    return (
+        <Box className="tw-w-fit">
+            <InlineDialog
+                open={isOpen}
+                minHeight={args.minHeight}
+                maxHeight={args.maxHeight}
+                minWidth={args.minWidth}
+                maxWidth={args.maxWidth}
+                handleClose={() => setIsOpen(false)}
+                modality={args.modality}
+                placement={args.placement}
+                flip={args.flip}
+                offset={args.offset}
+                enablePortal={args.enablePortal}
+                darkUnderlay={args.darkUnderlay}
+                autoHeight={args.autoHeight}
+            >
+                <InlineDialog.Trigger>
+                    <Button
+                        emphasis={ButtonEmphasis.Default}
+                        icon={<IconDotsVertical16 />}
+                        onClick={() => setIsOpen(!isOpen)}
+                    ></Button>
+                </InlineDialog.Trigger>
+                <InlineDialog.Content>
+                    <DialogBody>
+                        <Box className="tw-p-4">
+                            <Box className="tw-mb-2">
+                                <Flex justify="start">
+                                    <Tooltip
+                                        triggerElement={
+                                            <button aria-label="Exclamation mark circle icon" className="tw-mr-1">
+                                                <IconExclamationMarkCircle16 />
+                                            </button>
+                                        }
+                                        content="Just some Information"
+                                    />
+                                    <p>Information</p>
+                                </Flex>
+                            </Box>
+                            <Dropdown
+                                onChange={(id) => console.log(id)}
+                                activeItemId="1"
+                                menuBlocks={[
+                                    {
```

Comment:
All the menu items are disabled or show an incorrect pointer. Not sure if this is an issue with this implementation or if it's on the component itself?

Reply:
It seems that the pointer is based on the content underneath the dropdown menu somehow

Reply:
It's an issue when nested overlay components are using Portals apparently. I removed it from dropdown as it seems to not be necessary. But this next quarter, I'm going to heavily test interactions of overlays within overlays and the dropdown will also be replaced by the Select. So it's "To be continued..." topic.